src/snowflake/connector/network.py

```diff
@@ -689,7 +691,6 @@ def _get_request(
             headers,
             timeout=timeout,
             token=token,
-            socket_timeout=socket_timeout,
```

Comment:
we no longer set socket timeout in `_get_request`?

Reply:
It is never used before, and the previously default socket timeout gets set below in the call stack anyways.